samples/bluetooth/hci_ipc/nrf5340_cpunet_iso-bt_ll_sw_split.conf

```diff
@@ -5,7 +5,7 @@ CONFIG_ISR_STACK_SIZE=1024
 CONFIG_IDLE_STACK_SIZE=256
 CONFIG_MAIN_STACK_SIZE=512
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
-CONFIG_HEAP_MEM_POOL_SIZE=4096
+CONFIG_HEAP_MEM_POOL_SIZE=1024
```

Comment:
This should be reverted. 1024 is too small

Reply:
Yes, but since I just rebased after some dependency PRs were merged, I propose to keep this commit as-is and see if the CI is still OK, then remove it afterward. RAM usage in cpu_net has already shown to be a bit critical, so I suggest to change 1 thing at the time.

Reply:
> Yes, but since I just rebased after some dependency PRs were merged, I propose to keep this commit as-is and see if the CI is still OK, then remove it afterward. RAM usage in cpu_net has already shown to be a bit critical, so I suggest to change 1 thing at the time.

This won't be caught by CI. It will only be triggered when running on-target on the nRF5340 I believe :) But would be caught by the PTS tests we can trigger

Reply:
it won't be caught by (auto)PTS... as we have overlay in bttester for nrf5340 netcore which set this to ...7k

Reply:
ahh scratch that, it loos like it uses just defaults from hci_ipc sample now

Reply:
> ahh scratch that, it loos like it uses just defaults from hci_ipc sample now

Yeah, we just modified that :)